small fixes to support Tableau (#8)
coralogix/coralogix-jdbc: USAGE.md

   | Windows | C:\Program Files\Tableau\Drivers |
   | Mac     | ~/Library/Tableau/Drivers        |
   | Linux   | /opt/tableau/tableau_driver/jdbc |

> 3. Copy [coralogix.tdc](coralogix.tdc) file in `~/Documents/My Tableau Repository/Datasources`

Reviewer: ```suggestion
3. Copy the [coralogix.tdc](coralogix.tdc) file to `~/Documents/My Tableau Repository/Datasources`
```

Author: Fixed

   For more details consult [Tableau documentation](https://kb.tableau.com/articles/howto/using-a-tdc-file-with-tableau-server)

> 4. Create file `coralogix.properties` and add there line with your `apiKey`.

Reviewer: ```suggestion
4. Create a file named `coralogix.properties` and add an entry with your `apiKey`.
```

Author: Fixed
